book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] Please make sure the rental instance variable is also accessible like the title and the author in order to set up the has-many-through relationship correctly.